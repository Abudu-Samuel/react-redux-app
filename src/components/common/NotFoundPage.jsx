import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
  <div>
    <h1 className="text-center">Page not found</h1>
    <p className="text-center"><Link to="/">Home</Link></p>
  </div>
);

export default NotFoundPage;