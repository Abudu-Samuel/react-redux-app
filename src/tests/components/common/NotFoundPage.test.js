/* eslint-disable */
import NotFoundPage from '../../../components/common/NotFoundPage';

describe('<NotFoundPage />', () => {
  it('render routes in the app', () => {
    const wrap = shallow(<NotFoundPage />);
    expect(toJson(wrap)).toMatchSnapshot();
  });
});