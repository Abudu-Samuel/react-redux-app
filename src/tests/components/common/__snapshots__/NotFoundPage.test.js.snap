// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> render routes in the app 1`] = `
<div>
  <h1
    className="text-center"
  >
    Page not found
  </h1>
  <p
    className="text-center"
  >
    <Link
      replace={false}
      to="/"
    >
      Home
    </Link>
  </p>
</div>
`;
